modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/ConfigurationProviderToolingAdapter.java

```diff
@@ -210,7 +198,7 @@ private MetadataContext getMetadataContext(ClassLoader classLoader) {
   private interface WithConnectionProviderCallable<T> {
 
     T call(ConnectionProvider connectionProvider, ConnectionProviderModel connectionProviderModel)
-        throws ValueResolvingException;
+        throws org.mule.sdk.api.values.ValueResolvingException;
```

Comment:
I think that you do not need the fqdn here, just ValueResolvingException, since the import has changed.